### DISCUSSION TITLE
Questions regarding this project

### DISCUSSION THREAD

**Discussion by Raj2032** [tauri-apps/tauri#3107] [General]
Hi I have some questions regarding this project:
1. Does it allow me to write applications for Android as well?
2. If `1` is applicable, is it web view apps or native apps for Android?
3. For Windows/Linux, would I get native performance or not?

**Comment by amrbashir** [comment]
1. In the future, it will.
2. It will be an Android activity with a Webview inside it.
3. Performance will depend on your skills and how well you can optimize it. Think of VSCode as the best possible performance you can get.

**Reply by Raj2032** [reply]
For number `3`, if I were to compare to other GUI libraries that is not web related technologies, would they perform better than Tauri as Tauri is a web powered thing?

**Reply by FabianLars** [reply]
Depends on how good they are and how "good" you use them. General performance comparisons are always pretty difficult or kinda misleading.
But removing the whole web overhead gives you in many cases a good base for better performance, but again there are too many variables to really say "native"=faster.

**Reply by Raj2032** [reply]
Ok thanks mate
